packages/wonder-blocks-button/components/button.md

```diff
@@ -83,181 +131,325 @@ const styles = StyleSheet.create({
 ```js
 const {View} = require("@khanacademy/wonder-blocks-core");
 const {StyleSheet} = require("aphrodite");
 
 const styles = StyleSheet.create({
     row: {
-        display: "row",
+        flexDirection: "row",
     },
-    wideButton: {
-        width: 200,
+    button: {
+        marginRight: 10,
+    }
+});
+
+<View style={styles.row}>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        size="small"
+    >
+        Label
+    </Button>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        kind="secondary"
+        size="small"
+    >
+        Label
+    </Button>
+    <Button
+        style={styles.button}
+        onClick={(e) => console.log("Hello, world!")}
+        kind="tertiary"
+        size="small"
+    >
+        Label
+    </Button>
+</View>
+```
+
+Buttons can have an `href` or an `onClick` handler or both.
+
+Being able to use both is necessary to support marking conversions in A/B
+tests.  There is however no built-in facility for doing A/B testing in
+`wonder-blocks` itself.
+```jsx
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
     },
-    centerInDiv: {
-        display: 'block',
-        margin: '0 auto',
+    button: {
+        marginRight: 10,
+    }
+});
+
+<View style={styles.row}>
+    <Button
+        href="#button-1"
+        style={styles.button}
+    >
+        href
+    </Button>
+    <Button
+        kind="secondary"
+        onClick={(e) => console.log("Hello, world!")}
+        style={styles.button}
+    >
+        onClick
+    </Button>
+    <Button
+        kind="tertiary"
+        href="#button-1"
+        onClick={(e) => console.log("Hello, world!")}
+        style={styles.button}
+    >
+        both
+    </Button>
+</View>
+```
+
+Buttons can do client-side navigation if `clientNav` is `true`.
+```jsx
+const {StyleSheet} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {MemoryRouter, Route, Switch} = require("react-router-dom");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        alignItems: "center",
+    },
+    button: {
+        marginRight: 10,
+    }
+});
+
+// NOTE: In actual code you would use BrowserRouter instead
+<MemoryRouter>
+    <View style={styles.row}>
+        <Button testId="button" href="/foo" clientNav={true} style={styles.button}>
+            Click me!
+        </Button>
+        <Switch>
+            <Route path="/foo">
+                <View id="foo">Hello, world!</View>
+            </Route>
+        </Switch>
+    </View>
+</MemoryRouter>
+```
+
+Buttons can have a `style` props which supports width, position, margin,
+and flex styles:
+
+TODO(kevinb): link to front matter section describing the `style` prop in
```

Comment:
I wish there was a way to include non-visible comments. It makes for a very awkward documentation reading experience. I think it'd be better to put these into Jira issues instead.